File: Exercicios-Beecrowd/Python/1051.py
x=float(input())
if x <= 2000:
    print(f'Isento')
elif x>200 and x<=3000:
    print(f'R$ {(x-2000)*0.08:.2f}')
elif x>3000 and x <=4500:
    print(f'R$ {((x-3000)*0.18)+80:.2f}')
else:
    print(f'R$ {((x-4500)*0.28)+350:.2f}')

File: Exercicios-Beecrowd/Kotlin/1051.kt
import java.util.*

fun main(args: Array<String>) {

    val salario= readLine().toString().toDouble()
    var cota=0.0
    if (salario<=2000.00){
        println("Isento")

    }
    else if(salario>2000.00 && salario<3000.00){
        cota=(salario-2000.00)*0.08
        println("R$ ${"%.2f".format(cota).replace(",",".")}")
    }
    else if (salario>3000.00 && salario<4500.00){
        val pass1=salario-3000.00
        cota=(pass1*0.18)+((salario-pass1-2000.00)*0.08)
        println("R$ ${"%.2f".format(cota).replace(",",".")}")

    }
    else{
        val valor1=salario-4500.00
        val soma1=valor1*0.28
        val novo_salario=salario-valor1-3000.00
        val soma2=novo_salario*0.18
        val novo_salario1= salario-valor1-novo_salario-2000.00
        val soma3=novo_salario1*0.08
        cota=soma1+soma2+soma3
        println("R$ ${"%.2f".format(cota).replace(",",".")}")
    }

}